[PL-48751] DEL 82700 Release Notes (#6176)
harness/developer-hub: release-notes/delegate.md

---
title: Delegate release notes
sidebar_label: Delegate
tags: [NextGen, "Delegate"]

> date: 2024-04-09T10:00

Reviewer: ```suggestion
date: 2024-04-xxT10:00
```

> date: 2024-04-15T10:00

Reviewer: ```suggestion
date: 2024-04-16T10:00
```

Reviewer: @saifeemustafaqharness Please update the date in the metadata. This controls the RSS feed. Thanks.

## April 2024

> ### Harness version 1.33.x, Harness Delegate version 24.03.82700 <!--  April 09, 2024 -->

Reviewer: ```suggestion
### Harness version 1.33.x, Harness Delegate version 24.04.82703 <!--  April xx, 2024 -->
```

> ### Harness version 1.33.x, Harness Delegate version 24.04.82703 <!--  April 15, 2024 -->

Reviewer: ```suggestion
### Harness version 1.33.5, Harness Delegate version 24.04.82703 <!--  April 15, 2024 -->
```

Reviewer: 24.04.82707 is the delegate version in QA. Verify the release version with @bharatgoelharness.

Reviewer: Please add https://harness.atlassian.net/browse/PL-48781

#### Fixed issues

> - Slack channel notifications failed due to an error related to explicitly setting the "Host" header as "hooks.slack.com". We have removed the explicit "Host" header setting to support both Slack-specific webhook URLs and regular URLs, resolving the issue in this version. (PL-47914)

Reviewer: ```suggestion
- Slack channel notifications failed due to an error related to explicitly setting the Host header as `hooks.slack.com`. We have removed the explicit Host header setting to support both Slack-specific webhook URLs and regular URLs, resolving the issue. (PL-47914)
```

## April 2024

### Harness version 1.33.5, Harness Delegate version 24.04.82707 <!--  April 16, 2024 -->

#### Fixed issues

- Slack channel notifications failed due to an error related to explicitly setting the Host header as `hooks.slack.com`. We have removed the explicit Host header setting to support both Slack-specific webhook URLs and regular URLs, resolving the issue. (PL-47914)

- In SCIM, creating a new user with special characters in their name failed, preventing the user from being added to Harness and resulting in discrepancies in user group membership between the Identity Provider and Harness. The name of a user will be sanitized if it does not follow Harness naming conventions during user addition flows. (PL-47614)

> - Builds triggered by Bitbucket Server push events have correct date information in the build history. This issue occurred due to missing date information in the `commits` object returned by the Bitbucket Server API. (CI-11556, ZD-58798)

Reviewer: ```suggestion
- Builds triggered by Bitbucket Server push events had incorrect date information in the build history. This issue occurred due to missing date information in the `commits` object returned by the Bitbucket Server API. (CI-11556, ZD-58798)
```
